pkgs/by-name/ln/lnav/package.nix

```diff
@@ -48,10 +54,16 @@ stdenv.mkDerivation rec {
     readline
     sqlite
     curl
-    gpm
```

Comment:
removed temporarily to be able to build

Reply:
Please simply guard it behind `stdenv.isDarwin`.